docs/VMR-Feature-Band-Support-Work-Plan.md

```diff
@@ -0,0 +1,136 @@
+- **Distro partner previously-source-built update process** - Right now distro partners will build a given release by taking a set of previously SB artifacts (provided by MS or source-built by the distro partner), and pass these to the build script via --with-packages and --with-sdk. Going forward, this step will either be preceded by an explicit step to update the root `Version.Details.xml` with these packages, or this switch will imply such a step. Therefore, when the build is running, the previously source-built inputs and their versions are specified by the `Version.Details.xml` file, and NuGet will find those packages in the directory passed via --with-packages. Essentially, we are eliminating some implicit behavior.
+
+### Toolset
+
+❓ Open Question:
+A policy is needed which defines what toolset version each feature band branch can have. For example, should all feature band branches be required to use only the 1xx toolset? Can there be a mix of some using 1xx while others use a higher version?
+
+### Build
+
+* In addition to NuGet package, we also need to provide a way to have access to the runtime and ASP.NET Core artifacts (e.g. `dotnet-runtime-<version>-linux-x64.tar.gz`) from 2xx branches. This is necessary for the SDK to be able to bundle things together. This should be able to be done using https://ci.dot.net, just as it is done in product repo builds.
+* While the `scenario-tests` repo doesn't have differing code for each feature band, it will still need to exist in each of the VMR's feature band branches because it is built as part of the pipeline run and used to validate the product. This is ok since it is not part of the shipped binaries.
+
+❓ Open Question:
+There will need to be some way to detect when we are in one of these 2xx branches. What mechanism should be used for that? Maybe checking `packageVersion` of the sdk in `source-manifest.json`?
+
+#### Source-Only Builds
+
+For source-only builds, you're going to need to build all the source, including the source from the 1xx branch in order to produce a release from a 2xx branch.
+
+There are a variety of scenarios to consider:
+
+1. Has the distro maintainer produced any 1xx assets before from the previous release?
```

Comment:
Would it be possible to skip the building of the tools?  A distro maintainer only releasing 2xx sdk wouldn't need to build all of the tools.